Testing: staticcheck fails

As noted in PR #85: On current master branch `staticcheck` fails:

````
$ staticcheck ./...
internal/wglinux/configure_linux_test.go:126:14: netlink.Nested | 0 always equals netlink.Nested (SA4016)
internal/wglinux/configure_linux_test.go:355:12: netlink.Nested | 0 always equals netlink.Nested (SA4016)
internal/wglinux/configure_linux_test.go:379:12: netlink.Nested | 0 always equals netlink.Nested (SA4016)
internal/wglinux/configure_linux_test.go:405:12: netlink.Nested | 0 always equals netlink.Nested (SA4016)
internal/wglinux/configure_linux_test.go:430:12: netlink.Nested | 0 always equals netlink.Nested (SA4016)
internal/wglinux/configure_linux_test.go:454:12: netlink.Nested | 0 always equals netlink.Nested (SA4016)
internal/wglinux/configure_linux_test.go:479:12: netlink.Nested | 0 always equals netlink.Nested (SA4016)
internal/wglinux/configure_linux_test.go:504:12: netlink.Nested | 0 always equals netlink.Nested (SA4016)
````

````
$ staticcheck -version
staticcheck 2020.1.3
````

The solution seems simple, remove the bit-wise OR 0. I will work on a PR today fixing this.

--- a/internal/wglinux/configure_linux_test.go
+++ b/internal/wglinux/configure_linux_test.go
@@ -123,7 +123,7 @@ func TestLinuxClientConfigureDevice(t *testing.T) {
 					Type: netlink.Nested | wgh.DeviceAPeers,
 					Data: nltest.MustMarshalAttributes([]netlink.Attribute{
 						{
-							Type: netlink.Nested | 0,
+							Type: netlink.Nested,
 							Data: nltest.MustMarshalAttributes([]netlink.Attribute{
 								{
 									Type: wgh.PeerAPublicKey,
@@ -352,7 +352,7 @@ func TestLinuxClientConfigureDeviceLargePeerIPChunks(t *testing.T) {
 			Type: netlink.Nested | wgh.DeviceAPeers,
 			Data: nltest.MustMarshalAttributes([]netlink.Attribute{
 				{
-					Type: netlink.Nested | 0,
+					Type: netlink.Nested,
 					Data: nltest.MustMarshalAttributes([]netlink.Attribute{
 						{
 							Type: wgh.PeerAPublicKey,
@@ -376,7 +376,7 @@ func TestLinuxClientConfigureDeviceLargePeerIPChunks(t *testing.T) {
 			Type: netlink.Nested | wgh.DeviceAPeers,
 			Data: nltest.MustMarshalAttributes([]netlink.Attribute{
 				{
-					Type: netlink.Nested | 0,
+					Type: netlink.Nested,
 					Data: nltest.MustMarshalAttributes([]netlink.Attribute{
 						{
 							Type: wgh.PeerAPublicKey,
@@ -402,7 +402,7 @@ func TestLinuxClientConfigureDeviceLargePeerIPChunks(t *testing.T) {
 			Type: netlink.Nested | wgh.DeviceAPeers,
 			Data: nltest.MustMarshalAttributes([]netlink.Attribute{
 				{
-					Type: netlink.Nested | 0,
+					Type: netlink.Nested,
 					Data: nltest.MustMarshalAttributes([]netlink.Attribute{
 						{
 							Type: wgh.PeerAPublicKey,
@@ -427,7 +427,7 @@ func TestLinuxClientConfigureDeviceLargePeerIPChunks(t *testing.T) {
 			Type: netlink.Nested | wgh.DeviceAPeers,
 			Data: nltest.MustMarshalAttributes([]netlink.Attribute{
 				{
-					Type: netlink.Nested | 0,
+					Type: netlink.Nested,
 					Data: nltest.MustMarshalAttributes([]netlink.Attribute{
 						{
 							Type: wgh.PeerAPublicKey,
@@ -451,7 +451,7 @@ func TestLinuxClientConfigureDeviceLargePeerIPChunks(t *testing.T) {
 			Type: netlink.Nested | wgh.DeviceAPeers,
 			Data: nltest.MustMarshalAttributes([]netlink.Attribute{
 				{
-					Type: netlink.Nested | 0,
+					Type: netlink.Nested,
 					Data: nltest.MustMarshalAttributes([]netlink.Attribute{
 						{
 							Type: wgh.PeerAPublicKey,
@@ -476,7 +476,7 @@ func TestLinuxClientConfigureDeviceLargePeerIPChunks(t *testing.T) {
 			Type: netlink.Nested | wgh.DeviceAPeers,
 			Data: nltest.MustMarshalAttributes([]netlink.Attribute{
 				{
-					Type: netlink.Nested | 0,
+					Type: netlink.Nested,
 					Data: nltest.MustMarshalAttributes([]netlink.Attribute{
 						{
 							Type: wgh.PeerAPublicKey,
@@ -501,7 +501,7 @@ func TestLinuxClientConfigureDeviceLargePeerIPChunks(t *testing.T) {
 			Type: netlink.Nested | wgh.DeviceAPeers,
 			Data: nltest.MustMarshalAttributes([]netlink.Attribute{
 				{
-					Type: netlink.Nested | 0,
+					Type: netlink.Nested,
 					Data: nltest.MustMarshalAttributes([]netlink.Attribute{
 						{
 							Type: wgh.PeerAPublicKey,

